src/blocked.c

```diff
@@ -111,7 +111,8 @@ void blockClient(client *c, int btype) {
 void updateStatsOnUnblock(client *c, long blocked_us, long reply_us){
     const ustime_t total_cmd_duration = c->duration + blocked_us + reply_us;
     c->lastcmd->microseconds += total_cmd_duration;
-
+    if (server.latency_track_enabled)
+        updateCommandLatencyHistogram(&(c->lastcmd->latency_histogram), total_cmd_duration*1000);
```

Comment:
I know that we currently do not account the blocked_us in total_cmd_duration, but maybe we should explicitly set the sample to be c->duration+reply_us?

Reply:
@ranshid we're tracking `c->duration + blocked_us + reply_us` on this one. So all account for.